Albania/ALB_extract_microdata_excel_to_csv.py

```diff
@@ -3,27 +3,24 @@
 
 # COMMAND ----------
 
-from tqdm import tqdm
-import openpyxl
 import pandas as pd
+MIN_NUM_OF_ROWS = 829223
+YEAR_RANGE = (2010, 2022)
 
 COUNTRY = 'Albania'
 microdata_csv_dir = prepare_microdata_csv_dir(COUNTRY)
 filename = input_excel_filename(COUNTRY)
-disaggregated_data_sheets = ['Data_Expenditures', 'Data_Revenues']
+sheet_name = 'Data_Expenditures'
+csv_file_path = f'{microdata_csv_dir}/{sheet_name}.csv'
 
-for sheet in tqdm(disaggregated_data_sheets):
-    csv_file_path = f'{microdata_csv_dir}/{sheet}.csv'
-    df = pd.read_excel(filename, sheet_name=sheet, header=0)
-    # Handle null and unnamed columns
-    header = [col_name for col_name in df.columns if normalize_cell(is_named_column(col_name))]
-    df = df[header]
-    
-    # Remove rows where all values are null
-    df = df.dropna(how='all')
+df = pd.read_excel(filename, sheet_name=sheet_name)
 
-    # Normalize cells
-    df = df.applymap(normalize_cell)
+# Handle null and unnamed columns
+header = [col_name for col_name in df.columns if normalize_cell(is_named_column(col_name))]
+df = df[header]
+df = df.dropna(how='all')
+df = df.applymap(normalize_cell)
+df.to_csv(csv_file_path, index=False, encoding='utf-8')
 
-    # Write to csv
-    df.to_csv(csv_file_path, index=False, encoding='utf-8')
+assert len(df) >= MIN_NUM_OF_ROWS, f"Number of rows in sheet are less than expected {MIN_NUM_OF_ROWS}, got {df.shape[0]}."
+assert df['year'].between(*YEAR_RANGE).all(), f"Values in 'Year' column in sheet are not within the range {YEAR_RANGE}."
```

Comment:
This checks if all the years in the dataframe is within the expected year range. What would happen if a new year gets added? What would be the pros and cons in checking if all years expected are present in the dataframe instead?